spec/lcm/integration/shared_contexts_for_lcm.rb

```diff
@@ -45,27 +55,19 @@ def create_workspace_csv(workspaces, client_id_column)
 end
 
 shared_context 'provisioning brick' do
+  include_context 'brick'
   before(:all) do
-    config_path = ConfigurationHelper.create_interpolated_tempfile(
-      @config_template_path,
-      @test_context
-    )
-    params = JSON.parse(File.read(config_path))
-    @brick_result = GoodData::Bricks::Pipeline.provisioning_brick_pipeline.call(params)
+    @brick_result = GoodData::Bricks::Pipeline.provisioning_brick_pipeline.call($params)
     client_pids = @brick_result[:params][:synchronize].first[:to].map(&:pid)
     $client_projects = client_pids.map { |id| @prod_rest_client.projects(id) }
     pp @brick_result
   end
 end
 
 shared_context 'rollout brick' do
+  include_context 'brick'
   before(:all) do
-    config_path = ConfigurationHelper.create_interpolated_tempfile(
-      @config_template_path,
-      @test_context
-    )
-    params = JSON.parse(File.read(config_path))
-    @brick_result = GoodData::Bricks::Pipeline.rollout_brick_pipeline.call(params)
+    @brick_result = GoodData::Bricks::Pipeline.rollout_brick_pipeline.call($params)
     client_pids = @brick_result[:params][:synchronize].first[:to].map(&:pid)
```

Comment:
Similar code found in shared_contexts_for_lcm.rb:57, shared_contexts_for_lcm.rb:67 (mass = 78)